src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+                        System.out.println(autoIndent(count + "." + item));
+                        count++;
+                    }
+                    borderLine();
+                }
+                break;
+            case MARK:
+                int taskToMark = Integer.parseInt(inp.split(" ")[1]) - 1;
+                if (taskToMark >= TASKS.size()) {
+                    prettifyOut("Task does not exist!");
+                } else {
+                    TASKS.get(taskToMark).markAsDone();
+                    prettifyOut("Nice! I've marked this task as done:\n" + TASKS.get(taskToMark));
+                }
+                break;
+            case UNMARK:
+                int taskToUnmark = Integer.parseInt(inp.split(" ")[1]) - 1;
+                if (taskToUnmark >= TASKS.size()) {
+                    prettifyOut("Task does not exist!");
+                } else {
+                    TASKS.get(taskToUnmark).markAsUndone();
+                    prettifyOut("OK, I've marked this task as not done yet:\n" + TASKS.get(taskToUnmark));
+                }
+                break;
+            case TODO:
+                Task td = new ToDo(description);
+                TASKS.add(td);
+                printTaskAdded(td);
+                break;
+            case DEADLINE:
+                String[] deadline = description.split(" /by ");
+                Task dl = new Deadline(deadline[0], deadline[1]);
+                TASKS.add(dl);
+                printTaskAdded(dl);
+                break;
+            case EVENT:
+                String[] event = description.split(" /from ");
+                String[] timing = event[1].split(" /to ");
+                Task ev = new Event(event[0], timing[0], timing[1]);
+                TASKS.add(ev);
+                printTaskAdded(ev);
+                break;
+            case DELETE:
+                int taskToDelete = Integer.parseInt(inp.split(" ")[1]) - 1;
+                if (taskToDelete >= TASKS.size()) {
+                    prettifyOut("Task does not exist!");
+                } else {
+                    prettifyOut("Noted. I've removed this task:\n  " + TASKS.get(taskToDelete) +
+                            "\nNow you have " + (TASKS.size() - 1) + " tasks in the list.");
+                    TASKS.remove(taskToDelete);
+                }
+                break;
+            case BYE:
+                prettifyOut(GOODBYE);
+                return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * The main function.
+     *
+     * @param args command line arguments.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        boolean cont = true;
```

Comment:
Maybe we can rename `cont` to sound more like a boolean? Something like `toCont` or `toContinue`?